### PR TITLE
Allow content to be defined at question level

### PR DESCRIPTION
**Pull request by MIslam92**
### What is the context of this PR? 

In relation to and needed to pass PR: https://github.com/ONSdigital/eq-survey-runner/pull/1540

Trello: https://trello.com/c/7Pqn9TmB/1936-remove-html-within-schemas-xs

### How to review
Run locally against the related branch to check if it passes validator

### CODE REVIEW

File: schemas/questions/general.json [hunk 1]
```diff
@@ -6,6 +6,9 @@
       "id": {
         "$ref": "../common_definitions.json#/id"
       },
+      "content": {
+        "$ref": "../common_definitions.json#/content"
+      },
```
Comment (MebinAbraham) [inline schemas/questions/general.json]: Will this not be required/useful for other question types?

**Comment by MebinAbraham** [comment]
@shahi645 
Re-review please.

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FROM python:3.6
 
-RUN pip install pipenv==8.2.7
+RUN pip install pipenv==11.10.0
```
Comment (shahi645) [inline Dockerfile]: @MIslam92 Should't this be done in separate PR, so we can track it with our commits to master

**Comment by MIslam92** [comment]
@nealedj I've just ran this locally and its passed for all of them?

**Comment by nealedj** [comment]
Hmm I was running it locally - must have run it against master or something. Fine for me now.

**Comment by nealedj** [comment]
@MIslam92 ready for squashing.
